lib/data/entity.js

```diff
@@ -86,8 +88,10 @@ const parseSubmissionXml = (entityFields, xml) => new Promise((resolve, reject)
 // request data in the form of JSON.
 const extractEntity = (body, propertyNames, existingEntity) => {
   const bodyKeys = ['label', 'uuid', 'data'];
-  if (Object.keys(body).filter(k => !bodyKeys.includes(k)).length > 0)
-    throw Problem.user.invalidEntity({ reason: 'Unrecognized fields included in request.' });
```

Comment:
@lognaturel, correct me if I'm wrong, but from a glance, I think this is the only use of `Problem.user.invalidEntity` that mentions "fields" instead of "properties". So I think this PR will close #874.

Reply:
Yes! Ideas for making this more actionable in the other comment thread.